TTC.Deployment.AmazonWebServices/AwsConfiguration.cs

```diff
@@ -8,12 +9,15 @@ public class AwsConfiguration
         public string AssumeRoleTrustDocument { get; set; }
         public string IamRolePolicyDocument { get; set; }
         public string Bucket { get; set; }
+        public Role AssumedRole { get; set; }
         public string RoleName { get; set; }
         public RegionEndpoint AwsEndpoint { get; set; }
         public AwsProxy Proxy { private get; set; }
         public AWSCredentials Credentials { get; set; }
         public string ProxyHost { get { return Proxy == null ? null : Proxy.Host; } }
         public int ProxyPort { get { return Proxy == null ? -1 : Proxy.Port; } }
+
+        internal Role Role { get; set; }

```

Comment:
I'd rather like to know what is the responsibilities we want to capture with that role (starting deployment?), so that I can create a minimum access set of requirements for it and, from the opposite side, also use it as a clue to verify the code to see whether it indeed performs minimum actions for this role.